pypeclub/OpenPype
# Royal Render: Fix plugin order and OpenPype auto-detection

## Bug

This PR is fixing various issues with **rr_control** publishing:

- [x] **rr_control** collectors order
- [x] OpenPype path with autodetection.
- [x] Terminate publish process when context selector closes without picking
- [x] Fix Start/End frame collection for ftrack publishing

⚠️ **This needs updating venv - `clique` must be 1.6.x to support publishing in python 3**

## How to test

update Royal Render plugin and publish some renders. Closing context selector without choosing anything should by handled gracefully. Start and End frames of published files should match rendered.
